src/test/java/com/puppycrawl/tools/checkstyle/checks/DescendantTokenCheckTest.java

```diff
@@ -276,4 +276,11 @@ public void testMaxTokenTypeReverseOrder() throws Exception {
                 getPath("InputDescendantTokenLastTokenType2.java"), expected);
     }
 
+    @Test
+    public void testProperty() throws Exception {
+        final String[] expected = CommonUtil.EMPTY_STRING_ARRAY;
```

Comment:
`testProperty` is really vague, please keep a specific name.

Reply:
@Kevin222004  , please fix method name

Reply:
done

Reply:
Input file has values that are not default,

Reply:
I have make it simple testLimitedToken